src/how-to-signer.md

```diff
@@ -1 +1,111 @@
 # How to operate a signer
+As a Stacker, you are responsible for ensuring the accuracy and integrity of incoming sBTC transactions. To satisfy this requirement and continue to earn BTC rewards, you must operate your own Signer or delegate your signing power to a third-party Signer. Below is a step-by-step guide to setting up and operating a Signer either by using the [default Signer implementation](#setting-up-a-default-signer) or by [utilizing the Signer SDK](#custom-signer-implementation).
+
+## Prerequisites
+Rust. To install, please [follow these instructions](https://www.rust-lang.org/tools/install).  
+Git
```

Comment:
Nit: My preference would be to say they only need the signer binary installed. We could distribute it so they don't have to build it from source to install it.

Reply:
It would also be a nice option to support running signers through Docker.

Reply:
Rust is still required for doing a default implementation and for cargo install it is recommended to install rustup in order to obtain cargo as part of the package (best way to install it)